tracim/views/errors.py

```diff
@@ -0,0 +1,5 @@
+from hapic.error import DefaultErrorBuilder
+
+
+class Error(DefaultErrorBuilder):
```

Comment:
rename `Error` to `TracimErrorBuilder`

Reply:
I wasn't sure about the naming as this class is also a marshmallow schema visible in swagger ui.